doc/api/util.md

```diff
@@ -426,6 +429,10 @@ changes:
     objects the same as arrays. Note that no text will be reduced below 16
     characters, no matter the `breakLength` size. For more information, see the
     example below. **Default:** `true`.
+  * `sorted` {boolean|Function} If specified, the output for all object types
+    besides arrays and typed arrays is going to be stable. If set to `true` the
```

Comment:
I’m not sure it’s obvious what `stable` means? Maybe just explain what happens, i.e. that entries will be sorted before being printed?